HGCalAnalysis/test/exampleConfig.py

```diff
@@ -17,21 +17,21 @@
 from Configuration.AlCa.GlobalTag import GlobalTag
 process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase2_realistic', '')
 from FastSimulation.Event.ParticleFilter_cfi import *
-from RecoLocalCalo.HGCalRecProducers.HGCalRecHit_cfi import dEdX_weights as dEdX
+from RecoLocalCalo.HGCalRecProducers.HGCalRecHit_cfi import dEdX
 
 process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(10) )
 
 process.source = cms.Source("PoolSource",
-    # replace 'myfile.root' with the source file you want to use
+    # replace 'step3.root' with the source file you want to use
     fileNames = cms.untracked.vstring(
-        '/store/mc/PhaseIITDRFall17DR/SingleGammaPt100Eta1p6_2p8/GEN-SIM-RECO/noPUFEVT_93X_upgrade2023_realistic_v2-v1/90000/02F2B8A2-44AA-E711-BD55-7845C4FC38ED.root'
+        'file:step3.root'
```

Comment:
Is there a RelVal file or a file that e.g. you produced yourself in 10_6_x_pre2 that we could point to have the example configuration work "out-of-the-box"?

Reply:
it's ten singlegamma:
`/afs/cern.ch/user/f/fpantale/public/step3.root`

Reply:
Can you copy that to our `eos` area, e.g. to a directory called `validation_files` so that it won't get deleted and point to it in the `exampleConfig.py` (I currently can't do it myself since I'm on a train)?

Reply:
```
/eos/cms/store/cmst3/group/hgcal/CMG_studies$ mkdir validation
mkdir: cannot create directory 'validation': Operation not permitted
```

Reply:
Alright, you should definitely have someone give you those rights. I will copy the file there later today, will let you know.

Reply:
```shell
[clange@lxplus017]~% ls /eos/cms/store/cmst3/group/hgcal/CMG_studies/validation
step3.root
```

Reply:
modified, thanks!